### PR TITLE
Breathe new life into the refcard

### PR DESCRIPTION
**Pull request by vermiculus**
I'm open to suggestions / requests, of course.

### CODE REVIEW

**Comment by expez** [comment]
This looks sweet!

File: doc/cider-refcard.tex [hunk 1]
```diff
@@ -1,110 +1,125 @@
...
+  \item[C-c C-d C-r] cider-grimoire
+  \item[C-c C-d A] cider-apropos-documentation
+  \item[C-c C-d h] cider-grimoire-web
+  \item[C-c C-d a] \alternate{cider-apropos}
+  \item[C-c C-d j] \alternate{cider-javadoc}
+  \item[C-c C-d d] \alternate{cider-doc}
+  \item[C-c C-d r] \alternate{cider-grimoire}
+\end{keylist}
+
+\section{CIDER control}
```
Comment (bbatsov) [inline doc/cider-refcard.tex]: Looking at the cornucopia of keybindings here I don't think this title is a good idea. I guess some of the keybindings might be moved elsewhere.

**Comment by bbatsov** [comment]
We also have to update the pdf in the repo.

**Comment by Malabarba** [comment]
Looks very nice.

**Comment by vermiculus** [comment]
> We also have to update the pdf in the repo.

The PDF was updated in the same commit.

File: doc/cider-refcard.tex (hunk 1 above)
Comment (vermiculus) [inline doc/cider-refcard.tex]: Are you talking about `CIDER control`?

Comment (bbatsov) [inline doc/cider-refcard.tex]: Yes.

**Comment by bbatsov** [comment]
> The PDF was updated in the same commit.

Sorry about this, I guess I didn't read the diff carefully enough.

File: doc/cider-refcard.tex (hunk 1 above)
Comment (bbatsov) [inline doc/cider-refcard.tex]: P.S. I know this title existed before your changes, but I feel this might be a good time to fix this section.

Comment (vermiculus) [inline doc/cider-refcard.tex]: It seems that most of this section is controlling the REPL (save `cider-selector`, I think). If the functions are going to be split up, where should they go? How should they be organized?

Comment (bbatsov) [inline doc/cider-refcard.tex]: I can argue only a handful of the commands here actually control the REPL (e.g. quit, refresh, interrupt). Tracing is a debugging feature. Maybe the commands that don't fit anywhere can go to some misc section.

**Comment by bbatsov** [comment]
P.S. A few commands were renamed recently, so this has to be accounted here as well.

File: doc/cider-refcard.tex (hunk 1 above)
Comment (cichli) [inline doc/cider-refcard.tex]: I think that the two Tracing commands should go under a new `Tracing` subsection of the `Programming` section, then the `CIDER control` section can be split up into these subsections:

* `REPL` - `cider-insert-last-sexp-in-repl` / `cider-switch-to-repl-buffer` / `cider-find-and-clear-repl-buffer` / `cider-repl-set-ns`

* `Connection` - `cider-display-current-connection-info` / `cider-rotate-connection` / `cider-quit` / `cider-restart` (this one is missing right now)

`cider-interrupt` and `cider-refresh` should then go into the `Evaluation` subsection of the `Programming` section (arguably we could split the `cider-load-` commands and `cider-refresh` into a seperate `Loading` subsection, but this is probably unnecessary).

`cider-selector` can go into `Navigation` maybe?

**Comment by bbatsov** [comment]
Ping :-)

**Comment by vermiculus** [comment]
Pingback :) I'll reorder according to @cichli's recommendation, but changes beyond that will require a deeper familiarity with CIDER's organization than I have right now. Clojure is still a hobby language for me :)

**Comment by bbatsov** [comment]
> Pingback :) I'll reorder according to @cichli's recommendation, but changes beyond that will require a deeper familiarity with CIDER's organization than I have right now.

Sounds good.

> Clojure is still a hobby language for me :)

Same for me & @Malabarba. :-)

**Comment by Malabarba** [comment]
Clearly, we need to start a Clojure consulting company and change this situation.

**Comment by vermiculus** [comment]
As soon as my student loans are paid off, count me in! :shipit:

**Comment by bbatsov** [comment]
@vermiculus Any progress here?

**Comment by bbatsov** [comment]
@vermiculus You have to rebase on top of the current `master`.

**Comment by vermiculus** [comment]
Will do

**Comment by vermiculus** [comment]
@bbatsov After a long struggle learning ediff and magit's rebasing stuff, I think we're all set :smile:

**Comment by bbatsov** [comment]
Thanks!

**Pull request merged**